src/frontends/pytorch/src/op/to.cpp

```diff
@@ -67,16 +68,25 @@ OutputVector translate_to(const NodeContext& context) {
     // memory format is ignored since it changes strides of a tensor. In openvino tensors are always contigious
     auto dtype_ext_node = context.get_input_from_visible_context(dtype_idx).get_node_shared_ptr();
     auto dtype_fw_node = ov::as_type_ptr<PtFrameworkNode>(dtype_ext_node);
+    auto data = context.get_input(0);
+
+    bool is_complex = as_type_ptr<ComplexTypeMark>(data.get_node_shared_ptr()) != nullptr;
+    if (is_complex)
+        data = data.get_node_shared_ptr()->input_value(0);
+
     Output<Node> cast;
     if (dtype_fw_node && dtype_fw_node->get_op_type() == "prim::dtype") {
         auto type_input = dtype_fw_node->input_value(0);
-        cast = context.mark_node(std::make_shared<v1::ConvertLike>(context.get_input(0), type_input));
+        cast = context.mark_node(std::make_shared<v1::ConvertLike>(data, type_input));
     } else if (const auto dtype_const = ov::as_type_ptr<v0::Constant>(dtype_ext_node)) {
         auto pt_type = dtype_const->cast_vector<int64_t>()[0];
         auto dtype = convert_dtype(pt_type);
-        cast = context.mark_node(std::make_shared<v0::Convert>(context.get_input(0), dtype));
+        cast = context.mark_node(std::make_shared<v0::Convert>(data, dtype));
     } else {
-        cast = context.mark_node(std::make_shared<v1::ConvertLike>(context.get_input(0), context.get_input(1)));
+        cast = context.mark_node(std::make_shared<v1::ConvertLike>(data, context.get_input(1)));
+    }
+    if (is_complex) {
+        cast = context.mark_node(std::make_shared<ComplexTypeMark>(cast, cast.get_element_type()));
     }
```

Comment:
complex type can be casted to real type. it works in PyTorch.
![{6C20B368-426A-4517-BF9F-B231126FC950}](https://github.com/user-attachments/assets/e1571d54-a4f3-4448-8dfd-f69493011478)

Please add tests